### PR TITLE
UX improvements in mesh command

### PR DESCRIPTION
**Pull request by pottekkat**
Signed-off-by: Navendu Pottekkat <navendupottekkat@gmail.com>

**Description**

This PR adds examples and removes the need for providing `--token` flag with the path to the token and instead uses the default token present in `.meshery` folder.

- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by navendu-pottekkat** [comment]
@leecalcote We should have this merged. Let's not leave it for after the release. There aren't any functional changes. Only UX changes to ensure consistency.

**Review by leecalcote — APPROVED** [review]

File: mesheryctl/internal/cli/root/mesh/deploy.go [hunk 1]
```diff
@@ -20,16 +21,30 @@ var (
...
+		Short: "Deploy a service mesh to the Kubernetes cluster",
 		Args:  cobra.MinimumNArgs(0),
-		Long:  `deploy service mesh in the connected kubernetes cluster`,
+		Long:  `Deploy a service mesh to the connected Kubernetes cluster`,
+		Example: `
+// Deploy a service mesh from an interactive on the default namespace
+mesheryctl mesh deploy
+
+// Deploy Linkerd mesh on a specific namespace
+mesheryctl mesh deploy --adapter linkerd --namespace linkerd-ns
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/mesh/deploy.go:32]: Should be `--adapter meshery-linkerd`

File: mesheryctl/internal/cli/root/mesh/deploy.go [hunk 2]
```diff
@@ -20,16 +21,30 @@ var (
...
+
+// Deploy Linkerd mesh and wait for it to be deployed
+mesheryctl mesh deploy --adapter linkerd --watch`,
```
Comment (leecalcote) [inline mesheryctl/internal/cli/root/mesh/deploy.go:35]: same

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by navendu-pottekkat** [comment]
@leecalcote Changes made.

**Review by piyushsingariya — APPROVED** [review]
We can merge this!

**Review by leecalcote — APPROVED** [review]
Good 👍

**Pull request merged**